G2P.pm

```diff
@@ -230,6 +230,14 @@ sub new {
     die("ERROR: Couldn't create log_dir $log_dir $!\n") if (!$return);
     $params->{log_dir} = $log_dir;
   } 
+  else{
+    opendir my $dh, $log_dir or warn "Not a directory";
```

Comment:
At this point we already tested if the directory exists. But there could still be a problem opening the directory. Could you please change `warn "Not a directory"` to `die("ERROR: There was a problem opening the log_dir: $!\n")`